File: src/main/java/io/sch/historyscan/infrastructure/features/codebase/exceptions/CloneCodeBaseException.java
package io.sch.historyscan.infrastructure.features.codebase.exceptions;

import io.sch.historyscan.domain.error.HistoryScanTechnicalException;

public class CloneCodeBaseException extends HistoryScanTechnicalException {
    public CloneCodeBaseException(String message, Throwable e) {
        super(message, e);
    }

    public CloneCodeBaseException(String message) {
        super(message);
    }
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/infrastructure/features/codebase/exceptions/CloneCodeBaseException.kt
package io.sch.historyscan.infrastructure.features.codebase.exceptions

import io.sch.historyscan.domain.error.HistoryScanTechnicalException

class CloneCodeBaseException : HistoryScanTechnicalException {
    constructor(message: String, e: Throwable) : super(message, e)

    constructor(message: String) : super(message)
}
